custom_components/ojmicroline_thermostat/icons.json

```diff
@@ -0,0 +1,21 @@
+{
+    "entity": {
+        "climate": {
+            "ojthermostat": {
+                "state_attributes": {
+                    "preset_mode": {
+                        "state": {
+                            "schedule": "mdi:clock-outline",
+                            "manual": "mdi:thermometer",
+                            "vacation": "mdi:umbrella-beach",
+                            "frost_protection": "mdi:snowflake",
+                            "boost": "mdi:rocket-launch",
+                            "comfort": "mdi:sofa",
+                            "eco": "mdi:leaf"
```

Comment:
Built-in presets use the built-in localization for strings but *not* icon translations, which I think is probably a bug. That's why we have to specify icons for built-in presets here.